aim/aim_manager.py

```diff
@@ -117,8 +127,6 @@ def update(self, context, resource, **update_attr_val):
         made to the database.
         """
         self._validate_resource_class(resource)
-        if not update_attr_val:
```

Comment:
Was this removed so that the sync status can be set to pending even with a no-op update?